geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -191,7 +191,10 @@ function Home({
         filtersFormItemsAllowed,
         theme,
         filters,
-        menu: {cfg: actionNavbarCfg} = {}
+        menu: {cfg: actionNavbarCfg} = {},
+        filtersForm: {cfg: {submitOnChange: filterFormsSubmitOnChange} = {}} = {},
+        filtersForm: {cfg: {debounceTime: filterFormstimeDebounce} = {}} = {}
```

Comment:
This kind of configuration should not contain the `cfg` key because is a specific property used by a plugin object configuration. In my opinion we don't need to make the `submitOnChange` and `debounceTime` configurable for now. We could have them as default props in FiltersForm for now.